### ISSUE TITLE
Failed to do git setup

### ISSUE THREAD

**Issue by mazmar** [WeblateOrg/weblate#1826]
In performance report this line keeps appearing. I've checked the chmod of the file and it is correct. Deleting the file does not help. 

Feb. 2, 2018, 6 a.m. | Git global setup | Failed to do git setup: error: could not lock config file /var/www/weblate/data/home/.gitconfig: File exists (255)

 * Weblate weblate-2.18-926-g5c0b9a1
 * Python 2.7.12
 * Django 1.11.6
 * six 1.10.0
 * social-auth-core 1.4.0
 * social-auth-app-django 1.2.0
 * django-appconf 1.0.1
 * Translate Toolkit 2.2.5
 * Whoosh 2.7.0
 * defusedxml 0.4.1
 * Git 2.7.4
 * Pillow (PIL) 1.1.7
 * dateutil 2.4.2
 * lxml 3.5.0
 * django-crispy-forms 1.7.0
 * compressor 2.2
 * djangorestframework 3.7.1
 * user-agents 1.1.0
 * pytz 2014.10
 * pyuca N/A
 * pyLibravatar N/A
 * PyYAML 3.11
 * Mercurial 3.7.3
 * Database backends: django.db.backends.postgresql_psycopg2

**Comment by nijel** [comment]
It's probably result of starting more processes in parallel and all of them doing setup. So you can ignore this warning, but it should be properly handled.

**Comment by nijel** [comment]
Thank you for your report, the issue you have reported has just been fixed. 

* In case you see problem with the fix, please comment on this issue. 
* In case you see similar problem, please open separate issue.
* If you are happy with the outcome, consider supporting Weblate by [donating](https://weblate.org/donate/).
